src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String color = new ColorSupplier().getRandomColor();
+        Ball ball = new Ball(color, (byte)new Random().nextInt(100));

```

Comment:
extract ColorSupplier and Random from here to class fields, same as in ColorSupplier class itself